src/main/scala/com/gu/contentapi/firehose/ContentApiFirehoseConsumer.scala

```diff
@@ -29,13 +29,11 @@ class ContentApiEventProcessor(filterProductionMonitoring: Boolean, override val
     event.eventType match {
 
       case EventType.Update | EventType.RetrievableUpdate =>
-        event.payload.foreach { payload =>
-          payload match {
+        event.payload.foreach {
             case EventPayload.Content(content) => streamListener.contentUpdate(content)
             case EventPayload.RetrievableContent(content) => streamListener.contentRetrievableUpdate(content)
-            case EventPayload.Atom(atom) => /* do nothing yet - TODO update */
+            case EventPayload.Atom(atom) => streamListener.atomUpdate(atom.asInstanceOf[Atom])
```

Comment:
I had to put a cast in here, as the instance `atom` is of a type EventPayloadAliases.AtomAlias, in the previous work with lambdas casting this way worked